cardano-node/src/Cardano/CLI/Tx/BenchmarkingTxSubmission.hs

```diff
@@ -0,0 +1,350 @@
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE LambdaCase          #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Cardano.CLI.Tx.BenchmarkingTxSubmission
+  ( ROEnv(..)
+  , RPCTxSubmission(..)
+  , TraceBenchTxSubmit(..)
+  , bulkSubmission
+  )
+where
+
+import           Prelude
+import           Cardano.Prelude (MonadIO, Word16, Seq, StateT
+                                 , evalStateT, gets, isJust, lift, {- liftIO, -} modify, toList, unless, when)
```

Comment:
Preludes are exempt from the explicit import list requirement -- so this one can be removed.

Reply:
Neil made it to avoid conflicts with `Prelude`.